sunpy/coordinates/frames.py

```diff
@@ -252,19 +252,23 @@ class HeliographicCarrington(BaseHeliographic):
     - The Z-axis (+90 degrees latitude) is aligned with the Sun's north pole.
     - The X-axis and Y-axis rotate with a period of 25.38 days.
 
-    This system differs from Stonyhurst Heliographic (HGS) in its definition of longitude.
+    This system differs from Stonyhurst Heliographic (HGS) in its definition of longitude.  This
+    longitude is an "apparent" longitude because it takes into account the time it takes for light
```

Comment:
I thought we were converting it to the "true" longitude, and not the apparent.

Reply:
Nope, the point of `HeliographicCarrington` is to do apparent longitudes for the specified observer.